.travis.yml

```diff
@@ -20,11 +20,4 @@ before_cache:
   - docker save -o docker/previous_image.tar.gz iot-lab-gateway
 
 script:
-  - >
-    docker run -v gateway_code:/home/iot-lab-gateway/gateway_code
-    -v $CWD/tests_utils/cfg_dir:/var/local/config
-    -e BOARD_TYPE=m3
-    -e CONTROL_NODE_TYPE=no
-    -e HOSTNAME=m3-00 
-    -e IOTLAB_GATEWAY_NO_INTEGRATION_TESTS
-    --privileged iot-lab-gateway-tests
+  - docker run iot-lab-gateway-tests
```

Comment:
if I remove tox from the dockerfile, it might be conterintuitive to run `docker run iot-lab-gateway-tests` (without the tox) and having the gateway rest server starting IMO

Reply:
I'm not sure the -t will change anythign for travis but I'll add it ^^

Reply:
Apparently it does make a difference (that's why I asked, I already noticed that ;) )